### ISSUE TITLE
outbounds calls to analytics.intelliscapesolutions.com

### ISSUE THREAD

**Issue by ramirezrick** [tomasf/caffeine#4]
What is the app so chatty to analytics.intelliscapesolutions.com? What information is being sent?

**Comment by miko-momox** [comment]
Exactly. It is sending on average 200-300 requests in a working day ( 8+ hours usage of the laptop ).

**Comment by Ekristoffe** [comment]
hello, could you share how do you have found this (what app have you use) I would like to do a test on my mac with older version to see when this problem have started and do a code check for what change has been done.
thanks

**Comment by ramirezrick** [comment]
> analytics.intelliscapesolutions.com

At the time I was using NextDNS, and I also saw it while using Pihole as well.
